contracts/vault-managers/VaultManagerChainlinkMainAsset.sol

```diff
@@ -285,29 +287,29 @@ contract VaultManagerChainlinkMainAsset is ReentrancyGuard {
     view
     {
         // main collateral value of the position in USD
-        uint mainUsdValue = oracle.assetToUsd(asset, vault.collaterals(asset, user));
+        uint mainUsdValue_q112 = oracle.assetToUsd(asset, vault.collaterals(asset, user));
 
-        _ensureCollateralization(asset, user, mainUsdValue);
+        _ensureCollateralization(asset, user, mainUsdValue_q112);
     }
 
     function _ensurePositionCollateralization_Eth(address user) internal view {
         // ETH value of the position in USD
-        uint ethUsdValue = oracle.ethToUsd(vault.collaterals(vault.weth(), user));
+        uint ethUsdValue_q112 = oracle.ethToUsd(vault.collaterals(vault.weth(), user)).mul(Q112);
```

Comment:
Why is it so? Which implementation is used here?

Reply:
The newer implementation is used here, see `ChainlinkOracleMainAsset_Mock` contract